### PR TITLE
Feature/project task board

### PR DESCRIPTION
**Pull request by riruADS**

### CODE REVIEW

File: app/controllers/api/react/api_controller.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
+class Api::React::ApiController < ActionController::Base
+  include Pundit
```
[the remark this answers is not in the document]
Comment (riruADS) [reply app/controllers/api/react/api_controller.rb]: ApplicationController has it, not ActionController::Base

File: app/controllers/api/react/sessions_controller.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
+class Api::React::SessionsController < Api::React::ApiController
+  skip_before_action :authenticate_user, only: [:create]
+  
+  # POST /api/react/sessions
+  def create
+    user = User.find_by(email: params[:email])
+    if user && user.valid_password?(params[:password])
```
[the remark this answers is not in the document]
Comment (riruADS) [reply app/controllers/api/react/sessions_controller.rb]: Not sure that .authenticate() will work, need to check it

File: config/routes.rb [hunk 3]
```diff
@@ -9,10 +9,26 @@
   # Root
   root to: 'public#index'
   get '/task-board', to: 'task_board#index'
```
[the remark this answers is not in the document]
Comment (riruADS) [reply config/routes.rb]: Yes, this string should be present

File: db/migrate/20191115091835_create_stacks.rb [hunk 4]
```diff
@@ -0,0 +1,10 @@
+class CreateStacks < ActiveRecord::Migration[5.2]
```
Comment (Skumring) [inline db/migrate/20191115091835_create_stacks.rb]: @riruADS you forgot to add `timestamps` fields, I will add this in my commit

**Comment by Skumring** [comment]
@riruADS reworked the feature for update `Stack` - https://github.com/appdev-academy/productive-team-rails/pull/8/commits/40a65099dc0ff1efb67b0430c5cb0edcc31d0aee

**Comment by Skumring** [comment]
Added lib `React-bootstrap`  let's use it to React App https://github.com/react-bootstrap/react-bootstrap --- https://react-bootstrap.github.io/

**Comment by Skumring** [comment]
At now I merged this large PR, @riruADS - to other adjustments/feature should be opened new PR

**Pull request merged**